### ISSUE TITLE
Upgrade from Chromium 103.0.5060.53 to Chromium 103.0.5060.66

### ISSUE THREAD

**Issue by emerick** [brave/brave-browser#23707]
Minor Chromium bump.

https://chromium.googlesource.com/chromium/src/+log/103.0.5060.53..103.0.5060.66?pretty=fuller&n=10000

QA tests:

Check branding items
Check for version bump

Additional checks:

No specific code changes in Brave (only line number changes in patches).

**Comment by kjozwiak** [comment]
Closing as the above was merged into `1.41.x` & `1.40.x` as per https://github.com/brave/brave-core/pull/13972 & https://github.com/brave/brave-core/pull/13969.

**Comment by GeetaSarvadnya** [comment]
Verification PASSED on

```
Brave | 1.40.109 Chromium: 103.0.5060.66 (Official Build) (64-bit)
-- | --
Revision | 20b1569438a85e631d15e83eb355e3e326e5da6f-refs/branch-heads/5060@{#1066}
OS | Windows 10 Version 21H2 (Build 19044.1766)
```
![image](https://user-images.githubusercontent.com/38657976/176104378-530645b1-9539-4bfc-82d2-95587c1ad516.png)
